tests/test_sourmash.py

```diff
@@ -159,199 +159,156 @@ def test_do_basic_compare_using_rna_arg(c):
     assert (cmp_out == cmp_calc).all()
 
 
-def test_do_compare_quiet():
-    with utils.TempDirectory() as location:
-        testdata1 = utils.get_test_data('short.fa')
-        testdata2 = utils.get_test_data('short2.fa')
-        status, out, err = utils.runscript('sourmash',
-                                           ['compute', '-k', '31',
-                                            testdata1, testdata2],
-                                           in_directory=location)
-
-
-        status, out, err = utils.runscript('sourmash',
-                                           ['compare', 'short.fa.sig',
-                                            'short2.fa.sig', '--csv', 'xxx',
-                                            '-q'],
-                                           in_directory=location)
-        assert not out
-        assert not err
+@utils.in_tempdir
+def test_do_compare_quiet(c):
+    testdata1 = utils.get_test_data('short.fa')
+    testdata2 = utils.get_test_data('short2.fa')
+    c.run_sourmash('compute', '-k', '31', testdata1, testdata2)
+
+    c.run_sourmash('compare', 'short.fa.sig',
+                   'short2.fa.sig', '--csv', 'xxx', '-q')
+    assert not c.last_result.out
+    assert not c.last_result.err
+
+# @utils.in_tempdir
```

Comment:
should this still be commented out?

Reply:
Oops! I don't remember commenting that out, must have been an accident, I'll fix it now.